projects/inspec.md

```diff
@@ -10,22 +10,32 @@ The InSpec project includes InSpec as well as several resource packs and plugins
 
 #### Project Owners
 
+James Stocks
+  - Github: [james-stocks](https://github.com/james-stocks)
+  - Slack: @Stocksy
+
+Nick Schwaderer
+ - Github: [Schwad](https://github.com/Schwad)
+ - Slack: @schwad
+
+#### Project Approvers
+
 Clinton Wolfe
   - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
   - Slack: @cwolfe
 
+#### Project Reviewers
+
+#### Project Alums
+
 Ryan Davis
   - Github: [zenspider](https://github.com/zenspider)
   - Slack: @rdavis
 
-#### Project Approvers
-
-#### Project Reviewers
-
 ### Project Repositories
 
 - [inspec](https://github.com/inspec/inspec)
-- [inspec-gcp](https://github.com/inspec/inspec-gcp)
+- [train](https://github.com/inspec/train)
```

Comment:
Train is a foundation project

Reply:
Oops - removed it from the list. Thanks for steering me right!